src/Symfony/Component/HttpKernel/Tests/HttpCache/HttpCacheTest.php

```diff
@@ -1147,8 +1147,33 @@ public function testEsiCacheSendsTheLowestTtl()
         $this->request('GET', '/', array(), array(), true);
         $this->assertEquals('Hello World! My name is Bobby.', $this->response->getContent());
 
-        // check for 100 or 99 as the test can be executed after a second change
-        $this->assertTrue(in_array($this->response->getTtl(), array(99, 100)));
```

Comment:
We probably don't need this workaround anymore since we've got clock mocking (agree @nicolas-grekas?)

Reply:
agreed :)